cypress/Shared/MeasuresPage.ts

```diff
@@ -79,8 +125,22 @@ export class MeasuresPage {
     public static validateVersionNumber(expectedValue: string, versionNumber: string): void {
         cy.readFile('cypress/fixtures/measureId').should('exist').then((fileContents) => {
 
-            let element = cy.get('[data-testid=measure-action-'+ fileContents +']').parent()
+            let element = cy.get('[data-testid=measure-action-' + fileContents + ']').parent()
             element.parent().should('contain', expectedValue).children().eq(1).should('contain', versionNumber)
         })
     }
+
+    public static clickDraftforCreatedMeasure(): void {
+
+        cy.readFile('cypress/fixtures/measureId').should('exist').then((fileContents) => {
+            Utilities.waitForElementVisible('[data-testid=measure-action-' + fileContents + ']', 30000)
+            cy.get('[data-testid=measure-action-' + fileContents + ']').should('be.visible')
+            Utilities.waitForElementEnabled('[data-testid=measure-action-' + fileContents + ']', 30000)
+            cy.get('[data-testid=measure-action-' + fileContents + ']').should('be.enabled')
+            cy.get('[data-testid=measure-action-' + fileContents + ']').click()
+            cy.get('[data-testid=draft-measure-' + fileContents + ']').click()
+
+            //<button data-testid="draft-measure-63d2951d5a023e64c3f4d77a">Draft</button>
```

Comment:
don't think we need this comment

Reply:
Removed all un-necessary comments from the MeasurePage file